### ISSUE TITLE
Improve Documentation of ResolveInfo

### ISSUE THREAD

**Issue by dgoldstein0** [graphql-python/graphql-core-legacy#42]
As far as I can tell there's no documentation anywhere about what it's fields mean and how to use it, except for maybe 1 example of "how to get requsted field names" buried in another issue.  As a fairly core part of the graphql library it needs documentation.

**Comment by jhgg** [comment]
Hi, @dgoldstein0, 

Docs #14 hasn't been done for a while -- I haven't really had any time to do it. Additionally, it's not recommended that you use this library directly. I would look into using [graphene](https://github.com/graphql-python/graphene) which has legit documentation & community support or [graphql-epoxy](https://github.com/graphql-python/graphql-epoxy) which is kinda in the same state as this project in terms of docs :(

Either way, this is noted and planned when myself, or someone... eventually gets to #14.

**Comment by dgoldstein0** [comment]
well, I jumped over here because graphene doesn't document it either, hoping to find some clarity.  Basically I've had to do a bunch of reading the source and guessing what might be useful to try... and eventually accomplishing something.  Unfortunately the source (both here and in graphene) is pretty much devoid of docstrings and other real documentation.

Anyhow I would love to see docs for both projects.  It's very hard to pick up this stuff because the docs are so poor.
